test/test_ad7091r.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+hardware = "ad7091r-8"
+classname = "adi.ad7091rx"
+
+
+#########################################
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+@pytest.mark.parametrize("channel", [0])
+def test_ad7091rx_rx_data(test_dma_rx, iio_uri, classname, channel):
+    test_dma_rx(iio_uri, classname, channel)
```

Comment:
I'm not sure this is the appropriate way to test current ad7091r-2/-4/-8 support.
I've heard that raspberry pies can optimize repeated SPI transfers so that their data go through the DMA path. Though, the Linux driver doesn't currently support buffered readings.
No-OS ad7091r8 driver does support buffered readings, but I'm not aware of any HDL project that enables transfers going through DMA. So, I think this test will not succeed in practice, unless `test_dma_rx()` can fall back to non-DMA transfers?

Reply:
`python -m pytest -vs -k ad7091rx --emu` throws `Exception: Buffer all zeros`. Maybe better to not assume the device is buffer capable in tests. Maybe check for buffer support prior to `test_dma_rx()`? Or maybe catch the exception?